app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php

```diff
@@ -638,6 +642,29 @@ public function resizeFile($source, $keepRatio = true)
         return false;
     }
 
+    /**
+     * Return width height for the image resizing.
+     *
+     * @param string $source
+     */
+    private function getResizedParams(string $source): array
+    {
+        $configWidth = $this->_resizeParameters['width'];
+        $configHeight = $this->_resizeParameters['height'];
+
+        //phpcs:ignore Generic.PHP.NoSilencedErrors
+        list($imageWidth, $imageHeight) = @getimagesize($source);
+     
+        if ($imageWidth && $imageHeight) {
+            $imageWidth = $configWidth > $imageWidth ? $imageWidth : $configWidth;
+            $imageHeight = $configHeight > $imageHeight ? $imageHeight : $configHeight;
+        } else {
```

Comment:
Honestly I'm not a fan of "else" statement where they can be avoided, what about
```
if ($imageWidth && $imageHeight) {
            $imageWidth = $configWidth > $imageWidth ? $imageWidth : $configWidth;
            $imageHeight = $configHeight > $imageHeight ? $imageHeight : $configHeight;
            return  [$imageWidth, $imageHeight];
}

return [$configWidth, $configHeight];

```

Reply:
ok